pages/search.html

```diff
@@ -0,0 +1,93 @@
+---
+title: Search results 
+permalink: /search/
+layout: layouts/base
+hide_nav_search: true
+---
+
+<script>
+//<![CDATA[
```

Comment:
what is this doing?

Reply:
From what I understand it helps with XML an XHTML parsing. I took the example from the cloud pages [jekyll template](https://github.com/cloud-gov/pages-uswds-jekyll/blob/8adeca42e14e72e6819e48f427e503502beddc23/search/index.html#L4)